packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -169,6 +167,21 @@ export async function checkVenueAccess(session: ISession, venue: string) {
   }
 }
 
+export async function checkForSubmissionUsingKey(session: ISession, venue: string, key: string) {
+  session.log.debug(`checking for existing submission using key "${key}"`);
+  try {
+    const submission = await getFromJournals(session, `sites/${venue}/submissions/by?key=${key}`);
```

Comment:
@stevejpurves we talked about maybe changing this to `/sites/${venue}/submissions?key=${key}`?